examples/features/effect-handlers.effekt.md

```diff
@@ -0,0 +1,87 @@
+# Effects and handlers
+
+For getting started with effects and handlers, we define the popular exception effect, consisting just of one effect operation `throw`:
+
+```
+interface Exception {
+  def throw(msg: String): Nothing
```

Comment:
We have a very flat subtyping lattice in Effekt. `Nothing` is bottom in this lattice and thus is a subtype of everything. However, there is no inhabitant of this type. Yet, it is still useful. It is commonly used as the return type for functions that do not return. This is exactly what happens in other languages with exceptions. `throw` will never return anything.

Reply:
Thanks for the question! We could also explain that somewhere ("it's the type for unreachable things") -- it might even be worthy to say that it's Rust's & TypeScript's `never`, C++'s `noreturn`.